pkg/karmadactl/cmdinit/cert/cert_test.go

```diff
@@ -107,6 +108,32 @@ func TestGenCerts(t *testing.T) {
 		t.Fatal(err)
 	}
 	if err := GenCerts(TestCertsTmp, TestCaCertPath, TestCaKeyPath, etcdServerCertConfig, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg); err != nil {
-		fmt.Println(err)
+		t.Fatal(err)
+	}
+
+	certFiles := []string{
+		"apiserver.crt", "apiserver.key",
+		"ca.crt", "ca.key",
+		"etcd-ca.crt", "etcd-ca.key",
+		"etcd-client.crt", "etcd-client.key",
+		"etcd-server.crt", "etcd-server.key",
+		"front-proxy-ca.crt", "front-proxy-ca.key",
+		"front-proxy-client.crt", "front-proxy-client.key",
+		"karmada.crt", "karmada.key",
+	}
+	if err := checkCertFiles(TestCertsTmp, certFiles); err != nil {
+		t.Fatal(err)
+	} else {
+		klog.Infof("All certificate files are present.")
+	}
+}
+
+func checkCertFiles(path string, files []string) error {
+	for _, file := range files {
+		filePath := fmt.Sprintf("%s/%s", path, file)
```

Comment:
I would prefer like this:

```suggestion
		filePath := filepath.Join(path, file)
```

Reply:
I have modified this code in the latest rebase, please check it again